tests/tensorflow/test_keras_pyfunc_model_works_with_all_input_types.py

```diff
@@ -289,6 +289,10 @@ def test_model_multi_multidim_tensor_input(
 
 
 @pytest.mark.parametrize("env_manager", ["local", "virtualenv"])
+@pytest.mark.skipif(
+    Version(tf.__version__) in [Version("2.16.2"), Version("2.17.0")],
+    reason="model concurrent loading fails due to issue #19927 in keras",
```

Comment:
```suggestion
    reason="model concurrent loading fails due to issue #19927 in keras",
```

Can we replace `#19927` with the URL?